docs/changelog.rst

```diff
@@ -5,6 +5,10 @@ Changelog
 =========
 
 
+- :release:`10.2.0 <21th August 2023>`
```

Comment:
This should be updated

Reply:
https://github.com/python-discord/bot-core/compare/b84ee8bc0b0652e415b61303ee53ae49d09141c7..a91c4546c77d10bb921b8e807631d375989e2fb3

Reply:
m0rge.